### PR TITLE
Add forums IDA

### PR DESCRIPTION
**Pull request by macdiesel**

### CODE REVIEW

File: docker-compose-sync.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ services:
   studio:
     volumes:
       - edxapp-sync:/edx/app/edxapp/edx-platform:nocopy
+  forums:
```
Comment (jmbowman) [inline docker-compose-sync.yml]: Putting on my DDD "ubiquitous language" hat: we have "forums" here, "forum" above, and our [glossary](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/glossary.html#f) redirects "forum" to "discussion" (and that documentation seems to have standardized on "discussion" throughout).  We should pick one and stick with it.

**Review by jmbowman — COMMENTED** [review]
I presume there will need to be some config changes in edx-platform to actually utilize this; is the plan to do that separately after this is merged?  And a minor nitpick, a couple of files are missing trailing newlines.

File: docker-compose-sync.yml (hunk 1 above)
Comment (macdiesel) [reply docker-compose-sync.yml]: Should be forum, forgot to change this one.

**Review by jmbowman — APPROVED** [review]

**Comment by SalahAdDin** [comment]
@macdiesel Don't forget add the properly repository for forums.

**Comment by macdiesel** [comment]
@SalahAdDin I'm not sure I follow?

**Comment by SalahAdDin** [comment]
It works very well, thank you!

**Pull request merged**